### PR TITLE
Update plotman.yaml

### PR DESCRIPTION
**Pull request by fihzy**
Add in the  pool_contract_address: example configuration.

### CODE REVIEW

**Comment by ericvitale** [comment]
@fihzy thanks for doing this. Waiting for this to get merged...

File: src/plotman/resources/plotman.yaml [hunk 1]
```diff
@@ -153,7 +153,8 @@ plotting:
         # Your public keys: farmer and pool - Required for madMAx, optional for chia with mnemonic.txt
```
Comment (altendky) [inline src/plotman/resources/plotman.yaml:153]:
```suggestion
        # Your public keys.  Be sure to use the pool contract address for
        # portable pool plots.  The pool public key is only for original
        # non-portable plots that can not be used with the official pooling
        # protocol.
```

**Review by altendky — COMMENTED** [review]
While we're here, let's add some commentary encouraging use of the pool contract address.  If you like this, you should be able to click the buttons to commit my suggestion right here in the web UI.

**Review by altendky — APPROVED** [review]
Thanks @fihzy, and welcome to the team!

**Comment by ericvitale** [comment]
How do I go about getting this change? I updated plotman again just now and it doesn't generate a config file with the new pooling options so I assume I don't have the latest. I ran the install command from the readme and then generated a config.

**Comment by altendky** [comment]
You can just add the parameter to your config file manually.  This change will be in "the next release".  Hopefully I'll do a small one "soon".

**Comment by ladiaz1028** [comment]
when I edit config file manually I get this:
 Traceback (most recent call last):
  File "/home/chia-plotter/chia-blockchain/venv/bin/plotman", line 8, in <module>
    sys.exit(main())
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/plotman/plotman.py", line 169, in main
    cfg = configuration.get_validated_configs(config_text, config_path, preset_target_definitions_text)
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/plotman/configuration.py", line 52, in get_validated_configs
    config_objects = yaml.load(config_text, Loader=yaml.SafeLoader)
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 4, column 1:
    version: [2]
    ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 172, column 9:
            type: chia
            ^

**Comment by altendky** [comment]
> when I edit config file manually I get this:
> Traceback (most recent call last):
> File "/home/chia-plotter/chia-blockchain/venv/bin/plotman", line 8, in
> sys.exit(main())
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/plotman/plotman.py", line 169, in main
> cfg = configuration.get_validated_configs(config_text, config_path, preset_target_definitions_text)
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/plotman/configuration.py", line 52, in get_validated_configs
> config_objects = yaml.load(config_text, Loader=yaml.SafeLoader)
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/**init**.py", line 114, in load
> return loader.get_single_data()
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/constructor.py", line 49, in get_single_data
> node = self.get_single_node()
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 36, in get_single_node
> document = self.compose_document()
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 55, in compose_document
> node = self.compose_node(None, None)
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 84, in compose_node
> node = self.compose_mapping_node(anchor)
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/composer.py", line 127, in compose_mapping_node
> while not self.check_event(MappingEndEvent):
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/parser.py", line 98, in check_event
> self.current_event = self.state()
> File "/home/chia-plotter/chia-blockchain/venv/lib/python3.8/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
> raise ParserError("while parsing a block mapping", self.marks[-1],
> yaml.parser.ParserError: while parsing a block mapping
> in "", line 4, column 1:
> version: [2]
> ^
> expected , but found ''
> in "", line 172, column 9:
> type: chia
> ^
> PLEASE HELP!

Please start a [new discussion](https://github.com/ericaltendorf/plotman/discussions/new) since this isn't related to development.  Be sure to include your full configuration file.  But, the error is saying you have invalid YAML.

**Pull request merged**